besu/src/main/java/org/hyperledger/besu/cli/DefaultCommandValues.java

```diff
@@ -57,7 +57,8 @@ public interface DefaultCommandValues {
   NatMethod DEFAULT_NAT_METHOD = NatMethod.AUTO;
   JwtAlgorithm DEFAULT_JWT_ALGORITHM = JwtAlgorithm.RS256;
   int FAST_SYNC_MIN_PEER_COUNT = 5;
-  int DEFAULT_MAX_PEERS = 25;
+  int DEFAULT_MAX_PEERS = 100;
+  int DEFAULT_P2P_PEER_LOWER_BOUND = 64;
```

Comment:
yes I want to collect some metrics on this. maybe 25/50 would be a more sensible default.

Reply:
I'd be very cautious about raising the peer limit.  From what I've seen besu works really well when it has 25 peers, it just tends to struggling to fill it's peer quota at times which is when it then struggles.  Experimenting with different values by overriding them in config is probably better than changing the defaults in code before we've checked what impact it will have.

Reply:
Indeed. I've changed the defaults to min = max = 25 which is equivalent to current behavior.